### PR TITLE
VACMS-5774: Add vc facility listing block to node view and form.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #5774

## Testing done
Visual / behat

## Screenshots
image
image

## QA steps
 - As an admin, go to `/admin/structure/types/manage/vet_center_locations_list/fields` and `/nashville-vet-center/locations` and compare to screenshots.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/field.field.node.vet_center_locations_list.field_nearby_vet_centers.yml [hunk 1]
```diff
@@ -18,8 +18,8 @@ id: node.vet_center_locations_list.field_nearby_vet_centers
 field_name: field_nearby_vet_centers
 entity_type: node
 bundle: vet_center_locations_list
-label: 'Nearby Vet Centers and Outstations'
```
Comment (kevwalsh) [inline config/sync/field.field.node.vet_center_locations_list.field_nearby_vet_centers.yml:21]: Wrong field! This field label change and addition of description should go on Nearby Mobile Vet Centers, not Nearby Vet Centers.

File: config/sync/field.field.node.vet_center_locations_list.field_nearby_vet_centers.yml [hunk 2]
```diff
@@ -18,8 +18,8 @@ id: node.vet_center_locations_list.field_nearby_vet_centers
...
-description: ''
+label: 'Display Additional Mobile Vet Centers'
+description: 'Add any additional Mobile Vet Centers that are not already associated with your Vet Center.'
```
Comment (kevwalsh) [inline config/sync/field.field.node.vet_center_locations_list.field_nearby_vet_centers.yml:22]: There's not enough padding between the description and the button. 

Edit_Vet_Center_-_Locations_List_Locations___VA_gov_CMS

File: config/sync/views.view.vet_center_facility_listing.yml [hunk 3]
```diff
@@ -744,6 +745,331 @@ display:
...
+          element_wrapper_class: ''
+          element_default_classes: true
+          empty: '<img src="/sites/default/files/2021-03/Placeholder%20-%20facility.jpeg" width="480" height="320" alt="Illustration of a facility with a flagpole" typeof="foaf:Image" class="image-style-thumbnail">'
+          hide_empty: false
+          empty_zero: false
+          hide_alter_empty: true
+          click_sort_column: target_id
+          type: media_thumbnail
+          settings:
+            image_style: 3_2_medium_thumbnail
```
Comment (kevwalsh) [inline config/sync/views.view.vet_center_facility_listing.yml:857]: Now that i see this in my place, i think we should make it look more like the entity browser fields below

- [ ] remove images from here. 
- [ ] remove facility status
- [ ] add Moderation state field, but change the label to "Status" to match the entity browser fields below. 
- [ ] Change label to "Main and satellite locatoins"
- [ ] Change fieldgroup to Details, not collapsed to match the ones below

Sorry for the change in tact!


Edit_Vet_Center_-_Locations_List_Locations___VA_gov_CMS

**Review by kevwalsh — CHANGES_REQUESTED** [review]

File: config/sync/field.field.node.vet_center_locations_list.field_nearby_vet_centers.yml (hunk 1 above)
Comment (ethanteague) [reply config/sync/field.field.node.vet_center_locations_list.field_nearby_vet_centers.yml:21]: Argh - sorry about that. They looked too similar to me in config, and I got my eyes crossed, @kevwalsh.

File: config/sync/field.field.node.vet_center_locations_list.field_nearby_vet_centers.yml (hunk 2 above)
Comment (ethanteague) [reply config/sync/field.field.node.vet_center_locations_list.field_nearby_vet_centers.yml:22]: Yeah, agreed, @kevwalsh - you were trying to tell me to do that in the ticket when you wrapped the description in a `p` !

File: config/sync/views.view.vet_center_facility_listing.yml (hunk 3 above)
Comment (ethanteague) [reply config/sync/views.view.vet_center_facility_listing.yml:857]: I was wondering the same - no problem, @kevwalsh :+1:

**Review by kevwalsh — APPROVED** [review]
Looks great Ethan

Edit_Vet_Center_-_Locations_List_Locations___VA_gov_CMS

File: config/sync/field.field.node.vet_center_locations_list.field_nearby_vet_centers.yml (hunk 1 above)
Comment (kevwalsh) [reply config/sync/field.field.node.vet_center_locations_list.field_nearby_vet_centers.yml]: no worries, it's the same problem that editors were facing too. Confusion.

**Review by kevwalsh — APPROVED** [review]
QA Pass, i checked as 5 different MVP users listed from https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/vet-centers/product-brief.md and edited Locations List page, all behaved as expected.

**Comment by kevwalsh** [comment]
It says there was a Behat failure but [i don't see any](https://tugboat.vfs.va.gov/log/60e4ca9e4fa3233927d8f332)

**Comment by ethanteague** [comment]
> It says there was a Behat failure but [i don't see any](https://tugboat.vfs.va.gov/log/60e4ca9e4fa3233927d8f332)

Looks like a connection issue - rebuilding.

**Pull request merged**